.github/workflows/pythonpackage_future3.yml

```diff
@@ -39,9 +39,9 @@ jobs:
         pip3 install -r src/jukebox/components/rfid/hardware/pn532_i2c_py532/requirements.txt
         pip3 install -r src/jukebox/components/rfid/hardware/rdm6300_serial/requirements.txt
         pip3 install -r src/jukebox/components/rfid/hardware/rc522_spi/requirements.txt
-    - name: Run pytest
+    - name: Run pytest with coverage
       run: |
-        ./run_pytest.sh --cov --cov-report xml
+        ./run_pytest.sh --cov --cov-report xml --cov-config=.coveragerc
```

Comment:
What do you expect by running it with coverage?

Reply:
In coveralls.io you can check how much of the code is covered by our unit tests.
The Python GitHub Action runs the tests with coverage.

See the coveralls comment in this PR or other out related PRs.

Reply:
OK, do we look for more coverage? Should we update the Contribution info as well?

Reply:
I think generally it's useful to improve the coverage, but from my point of view we should focus on making future3 feature "complete" at this point.

But it probably wouldn't hurt to encourage people in CONTRIBUTING to add unit tests.